### ISSUE TITLE
Insufficient color contrast in footer

### ISSUE THREAD

**Issue by bsmth** [mdn/fred#653]
### What information was incorrect, unhelpful, or incomplete?

The footer fails color contrast checks.

### What did you expect to see?

"Yes" button has sufficient color contrast. Other footer text has sufficient contrast to background.

### Do you have any supporting links, references, or citations?

https://pagespeed.web.dev/analysis/https-developer-mozilla-org-en-US-docs-Web-CSS-asin/vk1qcs5xz1?form_factor=mobile

### Do you have anything more you want to share?

This applies to light mode only.

The following passes with the current footer background color 

<img width="592" height="1062" alt="Image" src="https://github.com/user-attachments/assets/ff043291-b96d-4722-9949-1d5d06f73601" />

**Comment by caugner** [comment]
| Theme | Text | Background | Header |
|--------|--------|--------|--------|
| Light | #089141 | #EAF6EF | :x: [3.68:1](https://webaim.org/resources/contrastchecker/?fcolor=089141&bcolor=EAF6EF) |
| Dark | #70EBA3 | #0C311C | ✅ [9.58:1](https://webaim.org/resources/contrastchecker/?fcolor=70EBA3&bcolor=0C311C) | 

Note: With Firefox DevTools, the "Yes" button in the [Article footer](https://developer.mozilla.org/en-US/docs/Web/CSS#feedback)…

- … passes AAA with the Light theme:
  <img width="261" height="164" alt="Image" src="https://github.com/user-attachments/assets/9de4b883-7a8e-44d1-8702-a70dd7614638" />
- … fails AA with the Dark theme:
  <img width="251" height="147" alt="Image" src="https://github.com/user-attachments/assets/c3b2a8b6-e6b8-4057-8ee8-37d2469cc5f6" />

So it looks like DevTools doesn't use the right colors to determine the contrast ratio.
